### PR TITLE
1. enabled note content check and label content check. 2. added a clo…

### PR DESCRIPTION
**Pull request by Dave-114-P**
1. added a content and label counter. 2. added a close btn for label popup and slightly changed the css styling

### CODE REVIEW

File: app/static/css/add_note.css
Comment (csySally) [inline app/static/css/add_note.css:1]: image
the close button is not right

File: app/static/js/add_note.js [hunk 1]
```diff
@@ -32,7 +51,14 @@ document.querySelector("#btn-next").addEventListener("click", function () {
   console.log(wordCount);
   if (wordCount < 0) {
     alert("You can only type up to 60 words.");
```
Comment (csySally) [inline app/static/js/add_note.js:53]: would you please change it to "You can only type up to 100 words."

**Comment by Dave-114-P** [comment]
I have bettered the styling and alert details

**Comment by csySally** [comment]
Great work!

**Pull request merged**